### PR TITLE
Modify setVState method to use receiver identifier

### PR DESCRIPTION
**Pull request by vanou**
**What this PR does / why we need it**:
This commit modifies `setVState` method of `loggingT` to use receiver identifier.
Before this commit, this method doesn't use receiver but uses top-level variable `logging`.

**Which issue(s) this PR fixes** :
**None**

**Special notes for your reviewer**:
None

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by vanou** [comment]
Thanks @neolit123 for review. I added test function.

If you have a time, please review them & give comment.

**Comment by neolit123** [comment]
please have a look at the CI failures:
https://travis-ci.org/kubernetes/klog/builds/650417325?utm_source=github_status&utm_medium=notification

File: klog_test.go [hunk 1]
```diff
@@ -639,3 +639,73 @@ func TestInitFlags(t *testing.T) {
 		t.Fatal("Expected log_file_max_size to be 2048")
 	}
 }
+
+func createTestValueOfLoggingT() *loggingT {
+	l := new(loggingT)
+
```
Comment (neolit123) [inline klog_test.go]: all the empty lines in this function can be removed.

File: klog_test.go [hunk 2]
```diff
@@ -639,3 +639,73 @@ func TestInitFlags(t *testing.T) {
...
+	l.skipHeaders = false
+	l.skipLogHeaders = false
+	l.addDirHeader = false
+
+	return l
+}
+
+func createTestValueOfModulePat(p string, li bool, le Level) modulePat {
+	m := modulePat{}
+
```
Comment (neolit123) [inline klog_test.go]: ditto.

**Comment by vanou** [comment]
@neolit123 
Thank you for kind suggestion. I overlooked it.
And now I fix it.

File: klog_test.go (hunk 1 above)
Comment (vanou) [reply klog_test.go]: Thank you. I dealt with it.

File: klog_test.go (hunk 2 above)
Comment (vanou) [reply klog_test.go]: Thank you. I dealt with it.

**Comment by neolit123** [comment]
thanks
needs approval from the project approvers.
/lgtm

**Pull request merged**